contents/newsletter/engineers-do-support.md

```diff
@@ -0,0 +1,157 @@
+
+## How to scale engineers doing support
+
+As an engineer, you probably have one question in mind right now: "How does this scale?" Although we aren't a huge company, we're ~50 people supporting tens of thousands of users.
+
+We've done a lot of work to continue to [ship fast](https://newsletter.posthog.com/p/how-to-design-your-company-for-speed) while having engineers do support. Luckily for you, we're sharing what we've figured out here. 
+
+### 1. Prioritize
+
+As you get larger, the amount of support surpasses the amount of time you have to do support. All companies prioritize whether they say so or not. Making prioritization clear ensures your work aligns with company goals. At the moment, our support priority looks like this:
+
+1. Incidents
+2. Sales and customer success requests
+3. Priority organizations, trials, critical severity
+4. Paying customers, startups, billing issues, high severity
+5. Everything else
+
+We try our best to get to everything, but there are always going to be tickets you want to prioritize more than others. You should be clear about this and automate prioritization as much as possible.
+
+### 2. Support engineers
+
+As we grew, even dedicated support heroes weren't enough to keep up with the support load. They couldn't get through high-priority support, let alone the rest of the queue. This was causing them to neglect longer-term improvements critical for reducing support load.
+
+The fix for this was the introduction of [support engineers](/handbook/comms/customer-support) who are entirely dedicated to doing support. This role is not a traditional, non-technical, first-line support agent. They are expected to be able to figure out and fix issues, often by shipping code.
+
+A majority of support is still done by engineers, but support engineers solve as much as they can and escalate when needed. This helps support heroes work on the most pressing and valuable fixes.
+
+### 3. Community ≠ support
+
+For a long time, we had a Slack community. Although we had visions of it as being a place where users talked and learned about PostHog, it ended up functionally being dedicated to support. On top of this, knowledge was lost due to free plan limits, it was disconnected from our support queues, and it wasn't publicly searchable. 
+
+This led us to decide to [shut our Slack down](/blog/slack-closure).
+
+To replace it, we built a [community forum](/community) on our website. It is the opposite of Slack: permanent, public, customizable, and connected to our support queue. This enables us to provide a better support experience while also enabling us to craft a [real community experience](/handbook/community) for our users. 
+
+![Community forum](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_08_05_at_10_49_39_b1c6a55cfe.png)
+
+### 4. Make your product easier to support
```

Comment:
i reckon we could cut this point entirely? it feels a bit niche to what _we_ did and might not be widely applicable?